projects/GKv3/GKComponents/GKUI/Components/HyperView.cs

```diff
@@ -340,23 +340,51 @@ private void DoPaint(Graphics gfx)
             }
         }
 
-        private Font ProcessFont(Font prevFont, float emSize, FontStyle style)
+        private Font ProcessFont(Font prevFont, float emSize, FontStyle style, FontDecoration decoration)
         {
             Font result;
             if (prevFont == null) {
                 var defFont = this.Font;
-                result = new Font(defFont.FamilyName, emSize, style);
+                result = new Font(defFont.FamilyName ?? defFont.Family.Name, emSize, style, decoration);
             } else {
-                if (prevFont.Size == emSize && prevFont.FontStyle == style) {
+                if (prevFont.Size == emSize && prevFont.FontStyle == style && prevFont.FontDecoration == decoration) {
                     result = prevFont;
                 } else {
-                    result = new Font(prevFont.FamilyName, emSize, style);
+                    var familyName = prevFont.FamilyName ?? prevFont.Family.Name;
                     prevFont.Dispose();

```

Comment:
It seems that fonts with the same FontFamily share same handler. So disposing `prevFont` after creating result makes result invalid.

Reply:
8-) ok....